packages/core/src/fetch.ts

```diff
@@ -11,6 +11,13 @@ import { errorMessage } from "./error"
 import { logVerbose, roundWithPrecision, toStringList } from "./util"
 import { CancellationToken } from "./cancellation"
 import { readText } from "./fs"
+import { HttpsProxyAgent } from 'https-proxy-agent';
+
+
+// We create a proxy based on Node.js environment variables.
+const proxy = process.env.HTTPS_PROXY || process.env.HTTP_PROXY;
```

Comment:
are these common names? should it be GENAISCRIPT_HTTPS_PROXY / GENAISCRIPT_HTTP_PROXY?

Reply:
There are the ones used by Python and NodeJS as default env variables.
 But we can add more in the expression.